src/components/progress-tracker/progress-tracker.stories.tsx

```diff
@@ -77,9 +77,30 @@ ColorVariants.storyName = "Color Variants";
 export const DefaultLabelValue: Story = () => {
   return (
     <Box display="flex" flexDirection="column" alignItems="center">
-      <ProgressTracker mt={2} progress={15} currentProgressLabel="15%" />
-      <ProgressTracker mt={2} progress={50} currentProgressLabel="50%" />
-      <ProgressTracker mt={2} progress={100} currentProgressLabel="100%" />
+      <ProgressTracker mt={2} progress={15} showDefaultLabels />
```

Comment:
question: if we're having to set `showDefaultLabels` in most of these stories is it worth setting it so it's true by default? @harpalsingh may be a question for you tbh

Reply:
If it should remain false by default I think it's wroth making sure we have a story that is capture by chromatic that has no labels at all

Reply:
I added this prop to most stories so that we didn't encourage using this component without labels, and I do think having this as true by default would be ideal but will let you/Harpal decide on that. 

I think that the `screenReaderOnlyLabel` story I added should be capturing the component without labels anyways.